test/cql-pytest/test_aggregate.py

```diff
@@ -217,3 +222,21 @@ def test_avg_decimal_2(cql, table1, cassandra_bug):
 def test_reject_aggregates_in_where_clause(cql, table1):
     assert_invalid_message(cql, table1, 'Aggregation',
                            f'SELECT * FROM {table1} WHERE p = sum((int)4)')
+
+# Reproduces #13265
+# Aggregates must handle case-sensitive column names correctly.
+def test_sum_case_sensitive_column(cql, table2):
+    p = unique_key_int()
+    cql.execute(f'insert into {table2} (p, c, somecolumn, "SomeColumn", "OtherColumn") VALUES ({p}, 1, 1, 10, 100)')
+    cql.execute(f'insert into {table2} (p, c, somecolumn, "SomeColumn", "OtherColumn") VALUES ({p}, 2, 2, 20, 200)')
+    cql.execute(f'insert into {table2} (p, c, somecolumn, "SomeColumn", "OtherColumn") VALUES ({p}, 3, 3, 30, 300)')
+
+    assert cql.execute(f'select sum(somecolumn) from {table2} where p = {p}').one()[0] == 6
```

Comment:
Nice test. I would also try `sum(SomeColumn)` (without quotes and check that the result is also 6, not 60 or error.